unit_tests/CMakeLists.txt

```diff
@@ -49,7 +54,12 @@ add_executable(NimbleSM_Unit ${NIMBLE_UNIT_SOURCES})
 message(" * Link nimble library")
 target_link_libraries(NimbleSM_Unit nimble::nimble)
 message(" * Link gtest libraries")
-target_link_libraries(NimbleSM_Unit gtest gtest_main)
+if (NIMBLE_USE_LOCAL_GTEST)
```

Comment:
Needed  this change to get my build going again because 
- it seems the CMake minimum version might be off—I do not have access to the FetchContent_MakeAvailable command. And
- (2), for linking with internal projects and running on diverse systems, it's preferred to use an available system version of GTest.

Reply:
I think that's for the best -- we initially added the FetchContent for gtest because GTest has API breakages between versions but it's good to be able to disable that if needed